src/frontend/deployment_details.go

```diff
@@ -19,19 +19,7 @@ func init() {
 	go loadDeploymentDetails()
 }
 
-func initializeLogger() {
-	log = logrus.New()
-	log.Level = logrus.DebugLevel
-	log.Formatter = &logrus.JSONFormatter{
-		FieldMap: logrus.FieldMap{
-			logrus.FieldKeyTime:  "timestamp",
-			logrus.FieldKeyLevel: "severity",
-			logrus.FieldKeyMsg:   "message",
-		},
-		TimestampFormat: time.RFC3339Nano,
```

Comment:
**Issue:** Since `time` is no longer used, we can remove it.
The `frontend` is failing to build (in the `deployment-tests` GitHub check) because of this error:
```
./deployment_details.go:6:2: imported and not used: "time"
```